src/mainwindow.py

```diff
@@ -53,15 +68,16 @@ def start(self):
         if not self.session:
             self.session = aiohttp.ClientSession()
 
+        cachesize = int(self.config['cachesize'])
         # Run 10 times to get 10 cats
-        for i in range(10):
+        for i in range(cachesize):
             # initialize a dict of cat data
             cat_data = dict()
 
             # randomly make jumpscares happen, but not on the first image
-            if randint(1, 10) == 5 and i:
+            if randint(1, cachesize) == 5 and i:
                 # get a random number for an image
-                image_number = randint(1, 10)
+                image_number = randint(1, cachesize)
```

Comment:
this number needs to be 1 through 10 because we only have 10 jumpscare images. This would throw an error if the cache size is larger than 10.